src/datapump/commands/version_update.py

```diff
@@ -45,3 +45,10 @@ class RasterVersionUpdateParameters(StrictBaseModel):
 class RasterVersionUpdateCommand(StrictBaseModel):
     command: str
     parameters: RasterVersionUpdateParameters
+
+
+class CogAssetParameters(StrictBaseModel):
+    implementation: str
+    source_pixel_meaning: str
```

Comment:
This works, I think my preference would be to actually link it to tile set job and get the ID from there, but that might make it fairly complicated

Reply:
I think I'll leave things as they are.  Since `pixel_meaning` is unique, this seems to be the simplest way to link up the jobs, especially given one of the tile set jobs is a tile_set_parameters job, and the other is an aux_tile_set_parameters job.